### DISCUSSION TITLE
Community namespace

### DISCUSSION THREAD

**Discussion by Miista** [LuckyPennySoftware/AutoMapper#4209] [NA]
Is there a specific sub-namespace to use for community packages?

**Comment by lbargaoanu** [comment]
There is one package I know of https://www.nuget.org/packages/AutoMapper.Contrib.Autofac.DependencyInjection.

**Reply by jbogard** [reply]
AutoMapper.* is a reserved package name, and AutoMapper.Contrib.* is marked public in NuGet.org.
